tests/summarycode/test_score.py

```diff
@@ -31,7 +31,7 @@
 test_env.test_data_dir = os.path.join(os.path.dirname(__file__), 'data')
 
 
-def make_test_function(test_name, test_dir, expected_file, regen=False):
+def make_test_function(test_name, test_dir, expected_file, regen=True):
```

Comment:
Same as above: You need to revert this to "regen=False", the regen True is only for local regeneration and cannot be committed... otherwise the tests are testing nothing.
And you would need to commit the updated expected JSON test results too